lib/mail_catcher/web/application.rb

```diff
@@ -5,14 +5,12 @@
 require "uri"
 
 require "sinatra"
-require "skinny"
+require 'faye/websocket'
```

Comment:
Please follow the conventions in the surrounds:

```suggestion
require "faye/websocket"
```

Reply:
fixed